code/modules/modular_computers/file_system/programs/skill_tracker.dm

```diff
@@ -0,0 +1,68 @@
+/datum/computer_file/program/skill_tracker
+	filename = "skilltracker"
+	filedesc = "ExperTrak Skill Tracker"
+	category = PROGRAM_CATEGORY_MISC
+	program_icon_state = "generic"
+	extended_desc = "Scan and view your current marketable job skills."
+	size = 4
```

Comment:
This should probably cost 1/2 since it's barely anything and we shouldn't make people decide whether they want this or the space, especially when cargo/science apps are so expensive.

Reply:
Fair enough, currently there is only 1 app that costs less than 4 (SignalCommander at 2) and I don't think this is very standardized at all, but I'd be willing to make it only cost 2. Does that size make sense for everyone?

Reply:
Changed it to 2 since it's reasonable.